cypress/support/fragments/requests/requests.js

```diff
@@ -229,4 +241,50 @@ export default {
     }).as('getTags');
     cy.wait('@getTags');
   },
+
+  requestTypes: { PAGE: 'Page', HOLD: 'Hold', RECALL: 'Recall' },
+
+  selectHoldsRequestType() {
+    cy.do(Checkbox({ name: 'Hold' }).click());
+  },
+
+  selectPagesRequestType() {
+    cy.do(Checkbox({ name: 'Page' }).click());
+  },
+
+  selectRecallsRequestType() {
+    cy.do(Checkbox({ name: 'Recall' }).click());
+  },
+
+  verifyFilteredResults(requestType) {
+    const values = [];
+    cy.get('[data-row-index]').each($row => {
```

Comment:
work through interactors please

Reply:
@dmitriyfolio currently, there is no interactor to extract cell values from `MultiColumnList`  with variable rows. In other test we had to use this approach.